### ISSUE TITLE
No such version: `@pooltogether/v4-periphery @ 1.3.0-beta.1`

### ISSUE THREAD

**Issue by ctjlewis** [pooltogether/v4-testnet#28]
We load this repo in [Bubki](https://github.com/0xBubki/donate) via:

```js
// package.json dependencies
"@pooltogether/v4-testnet": "https://github.com/pooltogether/v4-testnet.git",
```

But started receiving this error after the latest change to this package's (`v4-testnet`) package.json:

```
Couldn't find any versions for "@pooltogether/v4-periphery" that matches "1.3.0-beta.1"
```

I assume this release was published and then unpublished. How do you recommend we resolve? 

Smart contract engineers presumably needed something in this repo that wasn't in latest release, but this new dependency broke the git dependency approach.

**Comment by kamescg** [comment]
Hi @ctjlewis I would recommend always pulling packages from NPM instead of Github.

This dependency was accidentally pushed to the `main` branch, which is why you are having issues with installation.

**Comment by PierrickGT** [comment]
Has been fixed in version 1.3.0 of [@pooltogether/v4-testnet](https://www.npmjs.com/package/@pooltogether/v4-testnet)
